engine/execution/computation/manager.go

```diff
@@ -178,7 +178,7 @@ func (e *Manager) ExecuteScript(
 
 			prepareLog := func() *zerolog.Event {
 
-				args := make([]string, 0)
+				args := make([]string, 0, len(arguments))
 				for _, a := range arguments {
 					args = append(args, hex.EncodeToString(a))
```

Comment:
does it matter from the performance perspective whether it is append or simply args[i] = ?

Reply:
Also, what does the `args[:]` below do?  Is it needed?

Reply:
if the slice is sized correctly, you probably won't notice a difference between the two